docs/ropsten-testnet-usage.md

```diff
@@ -0,0 +1,44 @@
+
+## Using Peggy with the Ropsten testnet
+
+### Setup
+
+Before you can use the Bridge with the Ropsten testnet, you'll need to add two environment variables to the configuration file at `testnet-contracts/.env`. Add MNEMONIC from your MetaMask account, this will allow you to deploy the contracts to the Ropsten testnet. Add INFURA_PROJECT_ID from your Infura account, this will allow you to start a Relayer service which listens for events on the Ropsten testnet.
```

Comment:
```suggestion
Before you can use the Bridge with the Ropsten testnet, you'll need to add two environment variables to the configuration file at `testnet-contracts/.env`. Add `MNEMONIC` from your MetaMask account, this will allow you to deploy the contracts to the Ropsten testnet. Add `INFURA_PROJECT_ID` from your Infura account, this will allow you to start a Relayer service that listens for events on the Ropsten testnet.
```

Reply:
https://github.com/cosmos/peggy/pull/104/commits/cff943a3b789116e3459f065a41a01e93b4fa73b